components/serializer.rst

```diff
@@ -700,8 +700,11 @@ deserializing objects::
     $serialized = $serializer->serialize(new Person('Kévin'), 'json');
     // {"customer_name": "Kévin"}
 
-Serializing Boolean Attributes
```

Comment:
This deeplink needs to work, so we need a manual anchor

Reply:
I'm not sure to understand why this precise deep link needs to work?

Reply:
Because it could be shared in the meantime:

serializer.html#serializing-boolean-attributes

If we merge an deploy, this anchor is not working anymore, instead we need to add it manually to keep it working

Reply:
Here you go 👍